golang/cosmos/package.json

```diff
@@ -5,8 +5,7 @@
   "parsers": {
     "js": "mjs"
   },
-  "main": "src/index.js",
-  "module": "src/index.js",
+  "main": "src/index.cjs",
```

Comment:
The `"parsers"` above can also be removed (anywhere you encounter them going forward). They were necessary for Endo to briefly (and badly) straddle compatibility with both RESM and NESM.